pipelines/common/ibmcloud-ibmc-block.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ibmc-block-custom-storageclass
+  labels:
+    app: ibmcloud-block-storage-plugin
+    chart: ibmcloud-block-storage-plugin-1.5.0
+    heritage: Tiller
+    release: release-name
+  resourceVersion: "1609"
+parameters:
+  billingType: hourly
+  classVersion: "2"
+  fsType: ext4
+  iopsPerGB: "2"
+  sizeRange: '[20-12000]Gi'
+  type: Endurance
+provisioner: ibm.io/ibmc-block
+reclaimPolicy: Delete
```

Comment:
I think we want to make this recylce instead of delete right?  If not it's just the same as what IBM cloud provides by default?

Reply:
I verified from this https://kubernetes.io/docs/concepts/storage/persistent-volumes/#recycle that the `Recycle` is deprecated. So should we use Retain instead ? I have not tried running with `Recycle.` Let me try using Recycle once and if it works I can update it.

Reply:
After verification, the provisioner `ibm.io/ibmc-block` does not allow reclaimpolicy as `Recyle` but only allows `Retain` or `Delete`. So we need to give a template mostly with `Delete` reclaim policy. Is that correct @kvijai82